packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -376,31 +378,76 @@ export const components: ThemeOptions["components"] = {
   MuiCheckbox: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <CheckIcon />,
```

Comment:
Great solve here -- I like that we don't have to inline any SVGs as background-image